ProjectShop/Shipping/tests/tests.py

```diff
@@ -0,0 +1,19 @@
+from django.test import TestCase
+from django.db.utils import IntegrityError
+
+from Shipping.tests.factories import *
+
+class ShippingTestCase(TestCase):
+	def setUp(self) -> None:
+		pass
+
+    def test_create(self):
+        shipping = ShippingFactory()
+        self.assertIsNotNone(shipping)
```

Comment:
It seems like there is an indentation issue. What is the purpose of the setUp method, which does nothing?
![Screenshot from 2021-10-04 14-10-50](https://user-images.githubusercontent.com/56480204/135841630-d86df8cc-bd73-482b-986b-317100e4b030.png)